cwms-data-api/src/test/java/cwms/cda/data/dao/location/kind/OutletDaoIT.java

```diff
@@ -82,22 +84,40 @@ static void setup() throws Exception {
         }, CwmsDataApiSetupCallback.getWebUser());
     }
 
+    static void deleteLocation(DSLContext context, String locId, String officeId) {
+        LocationsDaoImpl locationsDao = new LocationsDaoImpl(context);
+        try {
+            locationsDao.deleteLocation(locId, officeId, true);
+        } catch (NotFoundException ex) {
+            LOGGER.atFinest().withCause(ex).log("No data found for " + officeId + "." + locId);
+        }
+    }
+
+    static void storeLocation(DSLContext context, Location loc) throws IOException {
+        LocationsDaoImpl locationsDao = new LocationsDaoImpl(context);
+        try {
+            locationsDao.deleteLocation(loc.getName(), loc.getOfficeId(), true);
```

Comment:
why not just call the above function?

Reply:
Oh good point.  I think I made that method after this and just didn't think about it.  Updated, thank you